src/utils/reducers/cartReducer.js

```diff
@@ -0,0 +1,84 @@
+export const ADD_ITEM = 'ADD_ITEM';
+export const SET_ITEM_QUANTITY = 'SET_ITEM_QUANTITY';
+export const REMOVE_ITEM = 'REMOVE_ITEM';
+export const EMPTY_CART = 'EMPTY_CART';
+
+export const cartInitialState = {
+  cart: {
+    items: [],
+    count: 0,
```

Comment:
## Feel free to not implement only a suggestion
You can add here the property `total` which is the total amount of the cart, the benefit of having here, it's that the responsibility of doing the operation is on the reducer and not on the component.